### PR TITLE
Binary Search in python

### PR DESCRIPTION
**Pull request by pallavidhere23**
Issue #106 (binary search implementation) has solved here in python.

### CODE REVIEW

File: Code/Python/Binary_Search.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+			found = True
+		else:
+			if item < item_list[mid]:
+				last = mid - 1
+			else:
+				first = mid + 1	
+	return found
+
+a=[]
+x=int(input("Enter x"))
```
Comment (rudrakshi99) [inline Code/Python/Binary_Search.py:33]: add a space after x.

File: Code/Python/Binary_Search.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+print(x)
+for i in range(0,x):
+    a.append(int(input("Enter next no:")))
+    print("array :",a)
+
+b=int(input("Enter Number to be Search"))
```
Comment (rudrakshi99) [inline Code/Python/Binary_Search.py:39]: add space after Search

File: Code/Python/Binary_Search.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+				last = mid - 1
+			else:
+				first = mid + 1	
+	return found
+
+a=[]
+x=int(input("Enter x"))
+print(x)
+for i in range(0,x):
+    a.append(int(input("Enter next no:")))
```
Comment (rudrakshi99) [inline Code/Python/Binary_Search.py:36]: add space after no.

**Review by rudrakshi99 — CHANGES_REQUESTED** [review]

**Comment by pallavidhere23** [comment]
@rudrakshi99 I have made all the required changes. Please review it.

**Review by rudrakshi99 — APPROVED** [review]

**Comment by rudrakshi99** [comment]
update the list in the sorting_searching folder.

**Comment by pallavidhere23** [comment]
[https://drive.google.com/file/d/1wJBhJX-o5LmRhYhopYgzI--uwtkmjLRw/view?usp=drivesdk](url) its already been updated according to the instruction given.

**Comment by pallavidhere23** [comment]
@rudrakshi99  Its already been updated you can see the files changed section of this pull request. I have added the Binary Search in readme of sorting_searching folder

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
add comments in your code.

**Comment by pallavidhere23** [comment]
> add comments in your code.

@tarun26091999 I have added the comments ,Please review it.

**Comment by rudrakshi99** [comment]
Resolve the merge conflicts.

**Comment by pallavidhere23** [comment]
@rudrakshi99 , I have resolve the merge conflicts, kindly review it.

**Pull request merged**